src-opam/windows.ml

```diff
@@ -252,13 +280,24 @@ module Cygwin = struct
       "flexdll=" ^ flexdll_version;
     ]
 
-  let mingw_packages = [ "make"; "diffutils"; "mingw64-x86_64-gcc-core" ]
-  let msvc_packages = [ "make"; "diffutils" ]
+  let mingw_packages =
+    [ "make"; "diffutils"; "patch"; "mingw64-x86_64-gcc-core"; "git" ]
+
+  let msvc_packages = [ "make"; "diffutils"; "patch"; "git" ]
 
-  let ocaml_for_windows_packages ?cyg ?(extra = []) ?(version = "0.0.0.2") () =
+  let install_ocaml_for_windows ?cyg ?(version = "0.0.0.2") () =
     let packages =
-      "make" :: "diffutils" :: "mingw64-x86_64-gcc-g++" :: "vim" :: "git"
-      :: "curl" :: "rsync" :: "unzip" :: "patch" :: "m4" :: extra
+      [
+        "rsync";
+        "patch";
+        "diffutils";
+        "curl";
+        "make";
+        "unzip";
+        "git";
+        "m4";
+        "perl";
```

Comment:
Why does this require Perl? Dropping `vim` seems reasonable for CI images though it might surprise some users of the Windows images.

Reply:
Usually the perl dependency comes from git. Perl is listed in the installation instructions of [OCaml for Windows](https://fdopen.github.io/opam-repository-mingw/installation/). We'll drop it once we remove OCaml for Windows.
I'm inclined to remove vim and nano altogether from the CI images, but as it may break workflows we don't know about I won't do it. I thought that starting from scratch, the Windows images didn't need them.